### PR TITLE
AvatarData: Fix avatarDataByteArray max size computation

### PR DESCRIPTION
**Pull request by hyperlogic**
The previous size calculation did not take the presence of the hand controller section into account.
This could sometimes result in a buffer overrun of the network buffer as identified in a debug build.

https://highfidelity.atlassian.net/browse/BUGZ-526

### CODE REVIEW

File: libraries/avatars/src/AvatarData.h [hunk 1]
```diff
@@ -312,17 +321,8 @@ namespace AvatarDataPacket {
...
-        SixByteQuat leftHandRotation;
-        SixByteTrans leftHandTranslation;
-        SixByteQuat rightHandRotation;
-        SixByteTrans rightHandTranslation;
-    } PACKED_END;
-    static const size_t HAND_CONTROLLERS_SIZE = 24;
-    static_assert(sizeof(HandControllers) == HAND_CONTROLLERS_SIZE, "AvatarDataPacket::HandControllers size doesn't match.");
-
+        AVATAR_LOCAL_POSITION_SIZE +
+        HAND_CONTROLLERS_SIZE;
```
Comment (SimonWalton-HiFi) [inline libraries/avatars/src/AvatarData.h]: Are the grab joints also allowed for (sizeof (AvatarDataPacket::FarGrabJoints))?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19884/HighFidelity-Beta-PR15729-2fa1671.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19884/HighFidelity-Beta-Interface-PR15729-2fa1671.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19884/HighFidelity-Beta-PR15729-2fa1671.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19884/HighFidelity-Beta-Interface-PR15729-2fa1671.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19884/HighFidelity-Beta-PR15729-2fa1671.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19884/HighFidelity-Quest-Beta-PR15729-2fa1671.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19890/HighFidelity-Beta-PR15729-897e276.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19890/HighFidelity-Quest-Beta-PR15729-897e276.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19890/HighFidelity-Beta-PR15729-897e276.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19890/HighFidelity-Beta-Interface-PR15729-897e276.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19890/HighFidelity-Beta-PR15729-897e276.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19890/HighFidelity-Beta-Interface-PR15729-897e276.dmg

**Pull request merged**